### ISSUE TITLE
Adjust API to work with one transcription file instead of multiple Settings, Error, Transcript

### ISSUE THREAD

**Issue by FlrnDml** [shuffle-project/asr-api#21]
see the Schema on how our API and Runner interaction should work

**ToDo**

- [ ] Adjust the endpoints and transcription logic to create one transcription_object (object that handles all logic in the transcription process) and store it in the `data` folder.
- [ ] Create an http-endpoint that queues the currently pending transcription requests for the runners to know what to do (see #17)

![Bildschirmfoto 2023-11-23 um 16 04 18](https://github.com/shuffle-project/asr-api/assets/93883574/55add7cc-b782-44aa-8756-cbe703d6652b)

https://excalidraw.com/#json=RwExlM8H3QSK0CJuYDcdN,Yr09g0Ei20TcKChSKoMVKA

**Comment by FlrnDml** [comment]
<img width="1016" alt="Bildschirmfoto 2023-11-17 um 16 01 05" src="https://github.com/shuffle-project/asr-api/assets/93883574/6896aa45-18f2-4cdc-bc34-9bbff34726e6">

**Comment by torbenziegler** [comment]
@FlrnDml 

I took some time to tackle this ticket, however with the current architecture I do not find it possible to make it work.

Previously in the meetings, we discussed a status.json file is enough to hold the state, so I do not understand why we hold the transcriptions as objects and return the status that way (once we shut off the server, they are gone as well). As soon as the transcription starts, we lose access to those objects since its running in a parallel process.

The previous status file handling implementation was working fine. Therefore, I recovered some of the deleted code and added that behaviour back in. Status file is now being created on request arrival with initial values. However, I am yet to explore how to add the finished transcript into the status file and update the "status" and "end_time" fields. As from my understanding, the transcript is added to the transcriptions directory in transcribe_to_json transcribe() function, correct?

The method returns none, so we cannot handle it immediately after that step. My suggestion is to take the created transcript_id.wav.json file and merge it with the existing status file. Then we would have the final and single status file. But where in the code does this have to be done?
